test/service/benches/block_import.rs

```diff
@@ -0,0 +1,90 @@
+// This file is part of Cumulus.
+
+// Copyright (C) Parity Technologies (UK) Ltd.
+// SPDX-License-Identifier: Apache-2.0
+
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// 	http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+use criterion::{criterion_group, criterion_main, BatchSize, Criterion, Throughput};
+
+use sc_client_api::UsageProvider;
+
+use core::time::Duration;
+use cumulus_primitives_core::ParaId;
+
+use sc_block_builder::{BlockBuilderProvider, RecordProof};
+use sp_api::{Core, ProvideRuntimeApi};
+use sp_keyring::Sr25519Keyring::Alice;
+
+use cumulus_test_service::bench_utils as utils;
+
+fn benchmark_block_import(c: &mut Criterion) {
+	sp_tracing::try_init_simple();
+
+	let runtime = tokio::runtime::Runtime::new().expect("creating tokio runtime doesn't fail; qed");
+	let para_id = ParaId::from(100);
+	let tokio_handle = runtime.handle();
+
+	// Create enough accounts to fill the block with transactions.
+	// Each account should only be included in one transfer.
+	let (src_accounts, dst_accounts, account_ids) = utils::create_benchmark_accounts();
+
+	let alice = runtime.block_on(
+		cumulus_test_service::TestNodeBuilder::new(para_id, tokio_handle.clone(), Alice)
+			// Preload all accounts with funds for the transfers
+			.endowed_accounts(account_ids)
+			.build(),
+	);
+	let client = alice.client;
+
+	let mut block_builder = client.new_block(Default::default()).unwrap();
+	block_builder.push(utils::extrinsic_set_time(&client)).unwrap();
+	let parent_hash = client.usage_info().chain.best_hash;
+	let parent_header = client.header(parent_hash).expect("Just fetched this hash.").unwrap();
+	block_builder.push(utils::extrinsic_set_validation_data(parent_header)).unwrap();
+	let built_block = block_builder.build().unwrap();
+	runtime.block_on(utils::import_block(&client, &built_block.block, false));
```

Comment:
Dumb question. Why we require to first import this block? And why it requires to have a `set_time` + `set_validation_data` extrinsic?